frontend/src/v5/ui/routes/dashboard/projects/calibration/calibrationHeader/calibrationHeader.component.tsx

```diff
@@ -35,14 +35,15 @@ const STEPS = [
 
 export const CalibrationHeader = () => {
 	const history = useHistory();
-	const { teamspace, project, containerOrFederation } = useParams();
-	const { step, setStep, isStepValid, vector3D, drawingId, origin } = useContext(CalibrationContext);
+	const { teamspace, project, containerOrFederation } = useParams<ViewerParams>();
+	const { step, setStep, isStepValid, vector3D, vector2D, drawingId, origin } = useContext(CalibrationContext);
 	const selectedModel = FederationsHooksSelectors.selectFederationById(containerOrFederation)
 		|| ContainersHooksSelectors.selectContainerById(containerOrFederation);
 	const isLastStep = step === 2;
 
 	const getIsStepValid = () => {
 		if (step === 0) return !!(vector3D[0] && vector3D[1]);
+		if (step === 1) return !!(vector2D[0] && vector2D[1]);
```

Comment:
Vector2D does not seem to be reset when the user clicks to change the arrow, unlike in the 3D calibration. This means that if the user clicks to start a new arrow the continue button is still valid which will use the old values for the 2D vector.
Gif showing how for 3D calibration the continue buttons gets disabled until the arrow is complete. There is also a secondary bug here where the user can go back to step 1 whilst in the middle of calibrating the 2D vector so it is still enabled but the toolbar option is hidden.
![arrows](https://github.com/user-attachments/assets/fdb056ef-4439-4764-9dfd-3f8164ecab19)
In fact just going backwards after setting the 2D vector keeps the 2D calibration tool vector so you can change it whilst in the 3D step. Is this okay?